File: app/src/main/java/uk/co/catedroid/app/data/model/Note.java
package uk.co.catedroid.app.data.model;

import com.google.gson.annotations.SerializedName;

public class Note {
    private String number;
    private String title;
    private String type;
    private String size;
    private String loaded;
    private String owner;
    private String hits;
    private String url;
    private String filekey;
    @SerializedName("module_number") private String moduleNumber;
    @SerializedName("module_name") private String moduleName;

    public String getNumber() {
        return number;
    }

    public String getTitle() {
        return title;
    }

    public String getType() {
        return type;
    }

    public String getSize() {
        return size;
    }

    public String getLoaded() {
        return loaded;
    }

    public String getOwner() {
        return owner;
    }

    public String getHits() {
        return hits;
    }

    public String getUrl() {
        return url;
    }

    public String getFilekey() {
        return filekey;
    }

    public String getModuleNumber() {
        return moduleNumber;
    }

    public String getModuleName() {
        return moduleName;
    }
}

File: app/src/main/java/uk/co/catedroid/app/data/model/Note.kt
package uk.co.catedroid.app.data.model

import com.google.gson.annotations.SerializedName

class Note(val number: String, val title: String, val type: String, val size: String,
           val loaded: String, val owner: String, val hits: String, val url: String,
           val filekey: String,
           @field:SerializedName("module_number") val moduleNumber: String,
           @field:SerializedName("module_name") val moduleName: String) {

    override fun toString(): String {
        return "Note{" +
                "moduleNumber='" + moduleNumber + '\''.toString() +
                ", moduleName='" + moduleName + '\''.toString() +
                ", title='" + title + '\''.toString() +
                '}'.toString()
    }
}
